doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1036 @@
+      See :ref:`matter_lock_sample_wifi_thread_switching` in the sample documentation for more information.
+    * Wi-Fi low power configuration using Wi-Fi's :ref:`Legacy Power Save mode <ug_nrf70_developing_powersave_dtim_unicast>`.
+
+* :ref:`matter_light_switch_sample`:
+
+  * Added Wi-Fi low power configuration using Wi-Fi's :ref:`Legacy Power Save mode <ug_nrf70_developing_powersave_dtim_unicast>`.
+
+NFC samples
+-----------
+
+* Fixed an issue where NFC samples that use the NFC Reader feature returned false error code with value ``1`` during the NFC T4T operation.
+
+Wi-Fi samples
+-------------
+
+* Added the :ref:`wifi_sr_coex_sample` sample demonstrating Wi-Fi Bluetooth LE coexistence.
+
+* Updated:
+
+  * The :ref:`wifi_shell_sample` sample now uses the :ref:`lib_wifi_credentials` and :ref:`wifi_mgmt_ext` libraries.
+  * The :ref:`wifi_provisioning` sample now uses the :ref:`lib_wifi_credentials` and :ref:`wifi_prov_readme` libraries.
+
+* Removed nRF7002 revision A support.
+
+Other samples
+-------------
+
+* Added the :ref:`mqtt_sample` sample that supports Wi-Fi and LTE connectivity.
+
+* :ref:`esb_prx_ptx` sample:
+
+  * Added support for front-end modules and :ref:`zephyr:nrf21540dk_nrf52840`.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+* :ref:`pmw3360`:
+
+  * Updated by reducing log verbosity.
+
+* :ref:`paw3212`:
+
+  * Updated by reducing log verbosity.
+
+* :ref:`lib_bh1749`:
+
+  * Fixed an issue where the driver would attempt to use APIs before the sensor was ready, which in turn could make the application hang.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Removed the dependency on the :ref:`lte_lc_readme` library.
+
+Bluetooth libraries and services
+--------------------------------
+
+* Added the :ref:`nsms_readme` library.
+
+* Added the :ref:`wifi_prov_readme` library.
+
+* :ref:`mds_readme` library:
+
+  * Fixed URI generation in the :c:func:`data_uri_read` function.
+
+* :ref:`ble_rpc` library:
+
+  * Fixed a possible memory leak in the :c:func:`bt_gatt_indicate_rpc_handler` function.
+
+* :ref:`bt_le_adv_prov_readme` library:
+
+  * Added the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_STOP_DISCOVERABLE_ON_RPA_ROTATION` Kconfig option to drop the Fast Pair advertising payload on RPA rotation.
+
+  * Updated:
+
+	* Extended the :c:struct:`bt_le_adv_prov_adv_state` structure to include new fields.
+      The new :c:member:`bt_le_adv_prov_adv_state.rpa_rotated` field is used to notify registered providers about Resolvable Private Address (RPA) rotation.
+      The new :c:member:`bt_le_adv_prov_adv_state.new_adv_session` field is used to notify registered providers that the new advertising session is about to start.
```

Comment:
```suggestion
	* Extended the :c:struct:`bt_le_adv_prov_adv_state` structure to include new fields.
          The new :c:member:`bt_le_adv_prov_adv_state.rpa_rotated` field is used to notify registered providers about Resolvable Private Address (RPA) rotation.
          The new :c:member:`bt_le_adv_prov_adv_state.new_adv_session` field is used to notify registered providers that the new advertising session is about to start.
```

Reply:
This is correct as is, github diff not showing the indendation correctly.